pkg/apis/seedmanagement/doc.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Nit: Those copyrights for new files could be changed to 2021

Reply:
Done